tools/packaging/common/istio-iptables.sh

```diff
@@ -48,40 +48,119 @@ function usage() {
   echo '      outbound traffic (i.e. "*") is being redirected (default to $ISTIO_SERVICE_EXCLUDE_CIDR).'
   echo '  -k: Comma separated list of virtual interfaces whose inbound traffic (from VM)'
   echo '      will be treated as outbound (optional)'
+  echo '  -t: Unit testing, only functions are loaded and no other instructions are executed.'
   # shellcheck disable=SC2016
   echo ''
   # shellcheck disable=SC2016
   echo 'Using environment variables in $ISTIO_SIDECAR_CONFIG (default: /var/lib/istio/envoy/sidecar.env)'
 }
 
 function dump {
-    iptables-save
-    ip6tables-save
+echo dump
+#    iptables-save
+#    ip6tables-save
```

Comment:
Why update this?

Reply:
my bad I was testing it on my mac where I do not have iptables so had to disable it temporarily.